### ISSUE TITLE
可以參考的東西

### ISSUE THREAD

**Issue by cheng1999** [cheng1999/Student-Performance-Predict#1]
### Dataset: 
https://www.kaggle.com/impapan/student-performance-data-set
### Paper: 
https://repositorium.sdum.uminho.pt/bitstream/1822/8024/1/student.pdf
### 其它我朋友的實作參考:
資料統計(非模型) [HW_6.pdf](https://github.com/cheng1999/Student-Performance-Predict/files/6532704/HW_6.pdf)
[期末報告範例_許騫心戴佩涵-Ecoli.pdf](https://github.com/cheng1999/Student-Performance-Predict/files/6532739/_.-Ecoli.pdf)
[1.pdf](https://github.com/cheng1999/Student-Performance-Predict/files/6532741/1.pdf)
[2.pdf](https://github.com/cheng1999/Student-Performance-Predict/files/6532742/2.pdf)
[3.pdf](https://github.com/cheng1999/Student-Performance-Predict/files/6532743/3.pdf)
https://github.com/cheng0908/DataMining

### Keywords:
**資料降維：** 
- PCA

**Models:**
- Decision Trees (DT)
- Random Forests (RF)
- Neural Networks (NN)
- Support Vector Machines(SVM)

- adaboost
- xgboost catboost

1. Adaboost 
2. SVM 
3. logistic regression 
4. Randomforest 
5.CNN
Cen Yang sent 3 June at 21:50
Adaboost的衍生：
1. gradient boost
2. xgboost
3. catboost

### 其它：
- 閒的話 google colab 可以玩玩看

- 預測目標
1: 01 boolean
2: ABCDE grading
3: percentage
數據允許我們預測得更精準，
但是不知道自己可以到什麼程度所以可以嘗試1，2，3的方法來評估
但是2，3不一定成功

**Comment by cheng1999** [comment]
臣洋你資料預處理有經驗，所以做好了可以給我們參考先
